ppl/detail/models/ZeekEvent.ts

```diff
@@ -0,0 +1,26 @@
+import {zng} from "zealot"
+import {BrimEventInterface} from "./BrimEvent"
+
+export class ZeekEvent implements BrimEventInterface {
+  constructor(private r: zng.Record) {}
+
+  getRecord() {
+    return this.r
+  }
+
+  getTime() {
+    return (this.r.get("ts") as zng.Primitive).toDate()
+  }
+
+  getEndTime() {
+    if (this.r.get("_path").toString() !== "conn") return null
+    const dur = (this.r.get("duration") as zng.Primitive).toFloat()
+    if (!dur) return
+    const ts = (this.r.get("ts") as zng.Primitive).toDate()
+    return new Date(ts.getTime() + dur * 1000)
+  }
```

Comment:
This endTime is used only on the Zeek connection event. Now we can have the length of the connection be rendered to scale like so:

<img width="339" alt="image" src="https://user-images.githubusercontent.com/3460638/103565124-9d781e80-4e74-11eb-833a-e58250e81534.png">